fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraBinaryImpl.java

```diff
@@ -270,14 +276,20 @@ public String getFilename() {
     }
 
     @Override
-    public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator) {
-        return getFixity(idTranslator, getContentDigest(), getContentSize());
+    public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator,
+                               final String baseURL,
+                               final String userAgent,
+                               final EventBus eventBus) {
+        return getFixity(idTranslator, getContentDigest(), getContentSize(), baseURL, userAgent, eventBus);
     }
 
     @Override
     public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator,
                                final URI digestUri,
-                               final long size) {
+                               final long size,
+                               final String baseURL,
+                               final String userAgent,
+                               final EventBus eventBus) {
```

Comment:
@ajs6f, once the blanknode PR gets in, maybe putting core model types under container management would be an interesting project for you to tackle?